go/kbfs/libkbfs/folder_block_manager_test.go

```diff
@@ -685,6 +685,7 @@ func TestFolderBlockManagerCleanSyncCache(t *testing.T) {
 	var userName kbname.NormalizedUsername = "test_user"
 	config, _, ctx, cancel := kbfsOpsInitNoMocks(t, userName)
 	defer kbfsTestShutdownNoMocks(t, config, ctx, cancel)
+	config.vdebugSetting = "vlog2"
```

Comment:
Should this be a constant?

Reply:
Probably but that ship may have sailed.  It's over in libkb and probably not appropriate to change in this PR.